### DISCUSSION TITLE
blog/11ty-bundle-15/

### DISCUSSION THREAD

**Discussion by giscus** [bobmonsour/11tybundle.dev#5] [Comments]
# blog/11ty-bundle-15/

Welcome to Issue 15 of the 11ty Bundle.

https://11tybundle.dev/blog/11ty-bundle-15/

**Comment by brycewray** [comment]
Congratulations on adding giscus (and thanks as always for the plug)! You might still want to check how your CSS is handling giscus’s built-in styling, since some surrounding elements look a bit odd if the user is in dark mode; but, as you say, there’s always time for that later. Take your vacation, sir. ✈️ 🏝️

Also, you made a **very** wise choice of Fathom Analytics over Netlify’s analytics offering, which has been, um, a questionable choice for some years now. I hope you have the same good luck with it and its excellent team that I’ve had, which includes getting just enough clicks on my own referral link that it more than pays for itself.

**Reply by bobmonsour** [reply]
Thanks, Bryce! I do plan to tackle the CSS part as it bugs be a bit to not have the background be the same color as the overall page background. And, as you point out, in dark mode, it's particularly odd looking.

So far, my interactions with Fathom have been very positive. And they are sufficiently responsive, though I've only posed one technical question to them so far. I had a site where the analytics weren't working and it turns out that I had a CSP and I hadn't told it about Fathom.

**Reply by bobmonsour** [reply]
I decided to remove the giscus comments from the blog as yours was the only comment that I received in the nearly 3 months since I added them...so, in the interest of less complexity and less javascript. They're gone. Sad.

**Reply by brycewray** [reply]
> I decided to remove the giscus comments from the blog as yours was the only comment that I received in the nearly 3 months since I added them...so, in the interest of less complexity and less javascript. They're gone. Sad.

@bobmonsour Yeah, I get very little action on mine as well. I was mainly guilted into bringing them back. Long story. 🤔
